invenio_app_ils/ui/backoffice/src/pages/UserDetails/components/UserMetadata/UserMetadata.js

```diff
@@ -0,0 +1,57 @@
+import React, { Component } from 'react';
+import { Grid, Segment, List, Header, Label } from 'semantic-ui-react';
+import PropTypes from 'prop-types';
+
+export default class UserMetadata extends Component {
+  render() {
+    const data = this.props.userDetails.data;
+
+    let label;
+    if (data.active) {
+      label = (
+        <Label as="a" color="green" ribbon>
+          {' '}
```

Comment:
css instead? I personally find `{' '}` ugly, but maybe it is just me :)

Reply:
Why is that `{' '}`?

Reply:
thanks, I've missed this one somehow

Reply:
Nope its not just you, `{'  '}`  is just ugly :)